src/js/components/Button/BusyAnimation.js

```diff
@@ -15,17 +15,18 @@ const bounceDelayRule = css`
   animation: ${bounceDelay} 1.4s infinite ease-in-out both;
 `;
 
+/* When button is small size we need half the dot size to fit properly */
 const Dot = styled(Box)`
   background-color: currentColor;
-  width: 8px;
-  height: 8px;
+  width: ${(props) => (props.size.size === 'small' ? '4px' : '8px')};
```

Comment:
It's not necessarily that the button has prop `size="small"` but rather that if the label is really short, the width of the button is more narrow, causing the "dots" to squish.

I'm wondering if instead these "Dot"s should just have a flex={false}?

Reply:
If the dots have `flex={false}` and the label is really short would the Button size grow horizontally when it enters the busy state?

Reply:
aw I didnt check the label so even on a medium size button with a small label this is happening

Reply:
☝️ 
so yes when we have `flex={false}` then the dots are not in the correct space. We may need to check the size of the button if it is under 100px seems to be where it is happening then change the values. 

https://github.com/grommet/grommet/assets/42451602/cc8068be-aadb-4448-afff-796f6eb2a837